### DISCUSSION TITLE
Export pole figure to Beartex format

### DISCUSSION THREAD

**Discussion by YohanTeX** [mtex-toolbox/mtex#559] [Ask Anything]
Hello, all! 
Is there any way to export pole figure data interpreted from ODF data?
I want to export the pole figure as BEARTEX format ASCII file if possible.
Thank you in advance!

Yohan

**Comment by lfgmorales** [comment]
Hi


You may want to have a look in the documentation


https://mtex-toolbox.github.io/PoleFigureImport.html


in principle *.xpe or *.xpf coming from Beartex should work



cheers



Luiz


Luiz F. G. Morales
ScopeM & Structural Geology and Tectonics Group
ETH Zürich
________________________________

**Reply by YohanTeX** [reply]
Hello, Luiz!
 
I've already imported ODF file and plotted pole figure.
I wanna export data of the pole figure I plotted as .txt file containing angles and corresponding intensity.
Is there any way to export data of pole figure?
Thanks.

Yohan

**Comment by kilir** [comment]
Have you had a look at https://mtex-toolbox.github.io/PoleFigure.export.html?
Cheers,
Rüdiger
